src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java

```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
@sabaka Inputs should be a direct copy or move. We expect no changes to input as this will cause line numbers for violations to change. Plus since this is a new file we will not be able to see what was changed in the input and that it was a valid change.
All changes to inputs will be done in another issue.